packages/web/src/app/shared/component-documentation/ceg/code-generator/code-generator.component.ts

```diff
@@ -50,6 +52,14 @@ export class CodeGeneratorComponent implements OnInit {
     if (this.activeTabIndex > this.tabs.length - 1) {
       this.setActiveTab(this.tabs.length - 1);
     }
+
+    this.preferredLanguageService.preferredLanguage$.subscribe((value) => {
+      this.setTabIndex(value);
+    });
+  }
+
+  ngOnDestroy(): void {
+    this.languageSubscription.unsubscribe();
```

Comment:
Consider moving this into the `constructor` and using a `takeUntilDestroyed` instead of manually unsubscribing, I am now getting an error from this I believe 👀 
![image](https://github.com/3lvia/designsystem/assets/38653456/064a88f1-f3ac-4976-912c-b78064c692b0)